resources/galera_configuration.rb

```diff
@@ -27,7 +27,7 @@
 property :server_id,                             Integer,        default: 100
 property :wsrep_sst_method,                      String,         default: 'rsync'
 property :wsrep_sst_auth,                        String,         default: 'sstuser:some_secret_password'
-property :wsrep_provider,                        String,         default: '/usr/lib/galera/libgalera_smm.so'
+property :wsrep_provider,                        [String, nil]
```

Comment:
Instead of this, I recommend you do the following:
```ruby
property :wsrep_provider, String, default: lazy { default_libgalera_smm_path }
```
And then you can remove what you added below on lines 52 and 53.